### PR TITLE
done ticket 10

### PR DESCRIPTION
**Pull request by Nosajthe1**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -289,3 +289,57 @@ describe("9.GET /api/articles/:article_id/comments", () => {
...
+    .send({
+      username: "butter_bridge",
+      body: "Hi there dudes we all feeling good?",
+    })
+    .expect(400)
+    .then((res) => {
+      expect(res.body.msg).toBe("Invalid URL - passed invalid ID");
+    });
+ });
+
```
Comment (karlaferry) [inline __tests__/app.test.js:342]: Please add more error handling, for instance, what if the username doesn't exist in the database? Or what if the post body does not have the required properties?

File: models/articlesModels.js [hunk 2]
```diff
@@ -55,3 +55,20 @@ exports.articleIdWithComment = (id) => {
...
+  console.log(id)
+  console.log(newComment)
+  const { username, body }  = newComment
+  
+  return db
+  .query(
+  `INSERT INTO comments (author, body, article_id) VALUES ($1, $2, $3) RETURNING *;`,
+          [username, body, id]
+          ).then(({rows}) => {
+            console.log(rows)
```
Comment (karlaferry) [inline models/articlesModels.js]: Clear out console.logs please before sending a PR.

**Review by karlaferry — CHANGES_REQUESTED** [review]
Just a few more stuff to add!

File: __tests__/app.test.js (hunk 1 above)
Comment (Nosajthe1) [reply __tests__/app.test.js:342]: SHall do it now! Thanks karla!

**Pull request merged**